src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1614,8 +1616,12 @@ void Partitioner::decompressionCutOff(const std::string& func_name) {
             ->build();
 
         // ChatGLM (GPTQ) and New LLaMa-v2 patterns (Symmetric)
-        rewr.add_matcher<ov::npuw::patterns::SymmZP::DCOFFPassReshape1>(dcoff_mode, dcoff_type, std::ref(params_to))
-            ->build();
+        auto dcoffPassReshape1 = std::make_shared<ov::npuw::patterns::SymmZP::DCOFFPassReshape1>(dcoff_mode,
```

Comment:
do you think is it better to move config value retrieval here?

Reply:
Do you mean within the constructor? I believe it is not a good idea to retrieve in this constructor since we shouldn't impact the other patterns. If we retrieve within this constructor, the derived classes also get impacted.

Reply:
@AsyaPronina  is right here, it should be added to the base class (like you actually did) and in the constructor (which is derived for all particular patterns).